CMakeLists.txt

```diff
@@ -311,6 +311,7 @@ add_custom_target(submit-p2
 )
 
 set(P3_FILES
+        "src/include/execution/executors/copy_file_executor.h"
```

Comment:
should not include in submission files, because it will not be modified by students